app/controllers/DataSetController.scala

```diff
@@ -78,14 +78,15 @@ class DataSetController @Inject()(val messagesApi: MessagesApi) extends Controll
   def list = UserAwareAction.async { implicit request =>
     UsingFilters(
       Filter("isEditable", (value: Boolean, el: DataSet) =>
-        el.isEditableBy(request.identity) && value || !el.isEditableBy(request.identity) && !value),
+        for {isEditable <- el.isEditableBy(request.identity)} yield {isEditable && value || !isEditable && !value}),
```

Comment:
also here, why the `for` syntax?

Reply:
isEditableBy is now also returning a Fox, so the alternative to the `for-yield` would have been a `map` of some kind, and I find the `for` easier to read